azure/errors.go

```diff
@@ -33,18 +32,9 @@ func ResourceNotFound(err error) bool {
 	return hasStatusCode(err, http.StatusNotFound)
 }
 
-// ResourceConflict parses an error to check if its status code is Conflict (409).
-func ResourceConflict(err error) bool {
```

Comment:
This isn't strictly related to `go-autorest`, but `ResourceConflict` was unused.